kinto_fxa/authentication.py

```diff
@@ -60,7 +63,13 @@ def unauthenticated_userid(self, request):
             return None
         if authmeth.lower() != 'bearer':
             return None
-        return self._verify_token(token, request)
+
+        user_id, client_name = self._verify_token(token, request)
+
+        if client_name is None or client_name == 'default':
+            return user_id
+
+        return '{}-{}'.format(user_id, client_name)
```

Comment:
The doc at [1] suggests this should have an "fxa:" prefix, like "fxa:5998e2e2e81c40bdad6d08ac2773bf61-lockbox".

[1] https://docs.google.com/document/d/19Tg5AyRdHDP7z_J6kWM-JgiO6tB6FmUuEXUweENYZq8/edit?ts=5a155487#

Reply:
Yes this will be added by kinto.